.github/workflows/terraform_conflict.yml

```diff
@@ -0,0 +1,19 @@
+name: Terraform Conflict
+
+on:
+  workflow_dispatch: null
+  pull_request:
+    types:
+      - closed
+      - converted_to_draft
+      - labeled
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+
+jobs:
+  tacos-gha:
+    uses: getsentry/tacos-gha/.github/workflows/tacos_conflict.yml@emarks/allow-other-users
```

Comment:
Blocked by https://github.com/getsentry/tacos-gha/pull/161
Resolve once that's merged.